README.md

```diff
@@ -1,125 +1,171 @@
-## Sort Package.json
+# Sort Package.json
 
 [![Build Status](https://travis-ci.org/keithamus/sort-package-json.svg)](https://travis-ci.org/keithamus/sort-package-json)
 
-Pass it a JSON string, it'll return a new JSON string, sorted by the keys typically found in a package.json
+## CLI
 
-Pass it an object, it'll return an object sorted by the keys typically found in a package.json
+### Install
 
-```js
-JSON.stringify(sortPackageJson({
-  dependencies: {},
-  version: '1.0.0',
-  keywords: ['thing'],
-  name: 'foo',
-}), null, 2)
-/* string:
-{
-  "name": "foo",
-  "version": "1.0.0",
-  "keywords": [
-    "thing"
-  ],
-  "dependencies": {}
-}
-*/
-```
-
-### Install Permanently
-
-#### API
-```bash
-npm install --save-dev sort-package-json
-```
-
-#### CLI
 ```bash
 npm install --global sort-package-json
 ```
 
-### CLI Usage:
+### Usage
 
 ```bash
 $ cd my-project
 $ cat package.json
 {
-  "dependencies": {},
+  "dependencies": {
+    "sort-package-json": "1.0.0",
+    "sort-object-keys": "1.0.0"
+  },
   "version": "1.0.0",
-  "keywords": [
-    "thing"
-  ],
-  "name": "foo"
+  "name": "my-awesome-project"
 }
-$ npm i -g sort-package-json
+
 $ sort-package-json
-/my-project/package.json is sorted!
-$ cat package.json
+package.json is sorted!
+
+cat package.json
 {
-  "name": "foo",
+  "name": "my-awesome-project",
   "version": "1.0.0",
-  "keywords": [
-    "thing"
-  ],
-  "dependencies": {}
+  "dependencies": {
+    "sort-object-keys": "1.0.0",
+    "sort-package-json": "1.0.0"
+  }
 }
 ```
 
-`sort-package-json` also supports arguments - so you can give it a bunch of package.json file(s) to sort.
+CLI also supports multi file paths or [`glob`](https://github.com/sindresorhus/globby) - so you can give it a bunch of `package.json` file(s) to sort.
 
 ```bash
-$ sort-package-json packages/*/package.json
+$ sort-package-json "my-package/package.json" "other-package/package.json"
 
-$ sort-package-json my-package/package.json other-package/package.json
+$ sort-package-json "package.json" "packages/*/package.json"
 ```
 
 #### `--check` flag
 
 When you want to check if your files are sorted, you can run CLI with the `--check` flag (or `-c`). This will output a list of not sorted files, if any.
 
 ```bash
-$ sort-package-json **/package.json --check
+$ sort-package-json "**/package.json" --check
 
 # 5 matched files are sorted.
 
 
-$ sort-package-json **/package.json --check
+$ sort-package-json "**/package.json" --check
 # foo/package.json
 # bar/package.json
 
 # 2 of 5 matched files are not sorted.
 ```
 
-
-### Run (with npm@5.2+)
+### Run with npx (npm@5.2+ required)
```

Comment:
Could we move this section higher? I dont think installing this package is the core use-case. `npx sort-package-json` is the ideal way to use this.

Reply:
Moved to top of CLI section